xapian-core/backends/dbfactory_remote.cc

```diff
@@ -28,10 +28,16 @@
 
 #include <string>
 
+#ifdef HAVE_FORK
+# include <signal.h>
+#endif
+
 using namespace std;
 
 namespace Xapian {
 
+map<string, pid_t> uuid_to_pid;
+
```

Comment:
This is the wrong place to be tracking this sort of thing.  This should be handled entirely in the test harness rather than requiring modifications to the library.  Doing this here adds overhead for library users for no good reason.